eng/pipelines/common/global-build-job.yml

```diff
@@ -162,13 +163,21 @@ jobs:
         inputs:
           filePath: $(Build.SourcesDirectory)/eng/pipelines/mono/update-machine-certs.ps1
 
+    - ${{ if eq(parameters.isManualCodeQLBuild, true) }}:
```

Comment:
This one should be inside the isSourceBuild guard too - otherwise we end up with an  empty call on codeQL for sourcebuild runs if they ever define the flag (and since the VMR is coming, it's not crazy to thinks verticals will use the source build mechanism).

Reply:
Good catch! Thanks :)